src/hotspot/share/cds/metaspaceShared.cpp

```diff
@@ -911,6 +911,7 @@ void MetaspaceShared::initialize_runtime_shared_and_meta_spaces() {
   if (static_mapinfo != nullptr) {
     log_info(cds)("Core region alignment: " SIZE_FORMAT, static_mapinfo->core_region_alignment());
     dynamic_mapinfo = open_dynamic_archive();
+
```

Comment:
Please remove this added blank line.

Reply:
This blank line was present before my change, I just restored it.